### DISCUSSION TITLE
How to properly enable STARTTLS or SSL/TLS

### DISCUSSION THREAD

**Discussion by MauricioOnetto** [modoboa/modoboa#2601] [Help]
Hey guys
Great work, thank you

I would like to use some accounts from gmail and I need TLS or SSL enabled

Should I do from:  Admin > Parameters > Webmail ???
Or I need to edit some files like /etc/postfix/main.cf ???

Thank you

**Comment by tonioo** [comment]
@MauricioOnetto That's not something you can do with Modoboa. You should take a look at Thunderbird or any other email client.
